import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,167 @@
+package testutils
```

Comment:
The license header is missing. Shall we add that?

Reply:
Fixed with 680508410867201cf5f4a96486d372c218b1c522